.github/workflows/license-check.yml

```diff
@@ -0,0 +1,20 @@
+name: license-check
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - main
+  pull_request:
+jobs:
+  license-check:
+    strategy:
+      matrix:
```

Comment:
This should not be needed as this is a simple bash script and does not require a matrix of versions or os to run on

Reply:
Makes sense, will change.